addons/html_editor/static/src/utils/image_processing.js

```diff
@@ -186,7 +186,7 @@ export async function loadImageInfo(el, attachmentSrc = "") {
     }
 
     const srcUrl = new URL(src, docHref);
-    let relativeSrc = srcUrl.pathname;
+    let relativeSrc = decodeURIComponent(srcUrl.pathname);
```

Comment:
here this might be more correct to use decodeURI:

```suggestion
    let relativeSrc = decodeURI(srcUrl.pathname);
```
normally:

- to encode decode pathname (eg. `http://site/{pathname}?q=search`) we use encodeURI / decodeURI
- to encode decode query parameter (`http://site/pathname?q={search}`) we use encodeURIComponent / decodeURIComponent

in the end both will probably work (it's just that decodeURIComponent decode more escape code) so both version are probably equivalent